pages/declaration.js

```diff
@@ -130,8 +130,7 @@ class Declaration extends React.Component {
               service (optional)
             </Checkbox>
             <Paragraph linkRenderer={NewTabLinkRenderer}>
-              See [here](/pdfs/feedback) for details on how we will use your
-              data.
+              [Details on how we use your data.](/pdfs/feedback)
```

Comment:
Looks better without full stop in a link

Reply:
Done